syntax/text/lexer/src/Luna/Syntax/Text/Lexer/Grammar.hs

```diff
@@ -342,10 +352,13 @@ symmap = Vector.generate symmapSize $ \i -> let c = Char.chr i in if
     where handleColons      = handleReps  [BlockStart, Typed]
           handleDots        = handleReps  [Accessor  , Range, Anything]
           handleEqs         = handleReps  [Assignment, Operator "=="]
-          handleHash        = handleRepsM [pure Disable, lexComment, lexConfig]
+          handleHash        = handleRepsM [lexComment, pure Disable, lexConfig]
           handleReps        = handleRepsM . fmap pure
-          handleRepsM ts s  = fromJust (pure $ Unknown s) $ ts ^? ix (Text32.length s - 1)
-          handleOp    op s  = if (op == "<" || op == ">") && s == "=" then Operator (op <> s) else case s of
+          handleRepsM ts s  = fromJust (pure $ Unknown s) $
+              ts ^? ix (Text32.length s - 1)
```

Comment:
If its possible and doesnt break the layout I'd really prefer the following form:
```
          handleRepsM ts s  = fromJust (pure $ Unknown s)
                            $ ts ^? ix (Text32.length s - 1)
```